### ISSUE TITLE
Intro messaging

### ISSUE THREAD

**Issue by yatil** [w3c/wai-wcag-quickref#27]
On this wiki page I try to collect all the necessary information that should go into the “About this Quick Reference” sections on the top of the page. See the [current “How to Meet WCAG 2.0” introductory text](http://www.w3.org/WAI/WCAG20/quickref/#intro) for a reference on what we might want to have in there.
- About
  - Describe in a short sentence what the quickref is all about.
  - Describe the relationship to WCAG 2.0
  - Link to more explanatory information on WCAG and accessibility
- Customization
  - Information about the Filters tab (that could also be a (?) symbol inside the tab?!)
  - Short descriptions on how Tags and Filters work
- Sharing
  - Describe the “share” links on the page
  - Especially the (new) “Share this view“ button in the top status bar
- Contribution
  - Blurb about how to contribute to the project
  - Blurb about how to submit new techniques

**Comment by iadawn** [comment]
- How to use
  - Info on who it is useful for, how it could be used in process?
  - Might be part of the about - although may be lost in there

**Comment by bakkenb** [comment]
I think the "Tags & Filters" section could be renamed "How to Use" and expanded upon. Start with basic functionality of selecting, scrolling and expanding techniques. Then add the information on Tags and Filters. Then you could finish with resetting and reloading the page to start from scratch.

**Comment by AndrewArch** [comment]
Structure is good, but I do agree that a bit more instruction might be useful

**Comment by gheake-1** [comment]
I think the structure is very good but would agree it could be expanded with some more instruction.

**Comment by happywebdiva** [comment]
I too like "How to Use" for the section currently labelled "Tags & Filters."  

One random thought and that is that these days in an online context I associate the word "sharing" with social media.  So I rather expect that button to act like the New York Times "more" button.  Maybe it could be renamed "Link" or "Link to this view"?  That said, I defer to whatever the editors prefer.

**Comment by davidberman** [comment]
I agree with that the verb "share" should be left reserved for social media connotations.

**Comment by sharronrush** [comment]
While it is true that the Share icon is most often associated with social media, I wanted to make a couple of points about this:
1. We had extensive discussion in meetings from May-June about the Share link and the limits that are imposed by it being on the W3C site (not able to easily link to social media sites). Therefore we voted to make the permalink available as a specific URL and to use the Share icon. 
2. This is entirely consistent with the way the link is provided within all of the WAI Tutorials.

It is an unconventional treatment but not at all hard to understand. I think we should leave it for the sake of consistency with other recently developed resources.  Shawn was going to inquire about the current attitude toward social media links and if she finds more flexibility in her research perhaps revisit. For now, however, we should go with the established internal convention, imo.

**Comment by sharronrush** [comment]
Agree with the change to "How to Use" rather than "Tags and Filters"

I find the boxes of text hard to look at when displayed and the layout a bit confusing, but I understand that it is probably just my bias for plain text so please register as a mild question to others rather than an objection. My (mild) preference would be for a series of accordions.

The text of the link "this Github repository" is not super clear. Suggest change to "QuickRef GitHub repository."

Awkward phrasing: "Tags apply to the Success Criteria. When selecting a tag, only the criteria that are tagged with that tag are visible, the others are collapsed. You can select multiple tags." Especially if the section is to be renamed "How to Use," I suggest an different approach. How about something like:

The tabs on the side navigation indicate two ways to use this resource. Choose "Contents" to browse the full list of WCAG Principles, Guidelines, and Success Criteria (SC). Choose "Filters" to select from a menu of tags that are directly related to specific SCs. You may choose as few as one or as many tags as are appropriate for your task. The result will be to hide SCs that are unrelated and to show only those that are relevant to your filtered search.

**Comment by James-Green** [comment]
I agree with Sharron, the way we use "Share" is appropriate.

**Comment by davidberman** [comment]
Okay: you've convinced me that the use of "share" is likely best left as is. thank you.

On 2015-10-12 09:33, sharronrush wrote:

> While it is true that the Share icon is most often associated with social media, I wanted to make a couple of points about this:
> 1. We had extensive discussion in meetings from May-June about the Share link and the limits that are imposed by it being on the W3C site (not able to easily link to social media sites). Therefore we voted to make the permalink available as a specific URL and to use the Share icon.
> 2. This is entirely consistent with the way the link is provided within all of the WAI Tutorials.
> 
> It is an unconventional treatment but not at all hard to understand. I think we should leave it for the sake of consistency with other recently developed resources. Shawn was going to inquire about the current attitude toward social media links and if she finds more flexibility in her 
> research perhaps revisit. For now, however, we should go with the established internal convention, imo.
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/w3c/wai-wcag-quickref/issues/27#issuecomment-147414362.

## 

David Berman, RGD, FGDC LinkedIn http://www.linkedin.com/in/bermandavid Twitter @davidberman http://www.twitter.com/davidberman Facebook http://www.facebook.com/davidbberman Skype davidberman.com skype:davidberman.com?chat Google Plus 
https://plus.google.com/u/0/+DavidBermanCom/posts
David Berman Communications | berman@davidberman.com | @davidberman | blog http://www.designedgecanada.com/author/david-berman
+1-613-728-6777 | 340 Selby Avenue, Ottawa K2A 3X6

## High Level Advisor, United Nations | GDC ethics chair | Ico-D Sustainability chair | Carleton University Access Network chair

/Accessibility courses:/ Vancouver | Victoria | Europe | Ottawa
/Upcoming:/ Dublin | Toronto | Mexico City | Cyprus | Bahrain
Watch David on CBS http://www.wtoc.com/story/17588481/scad-plans-revitalization | Do Good book news: http://www.dogoodbook.com/ "Don't just do good design ... do good!"

This message may contain proprietary information. Unauthorized disclosure/copying/distribution of contents prohibited.

**Comment by vmmiller** [comment]
Structure is good.  I didn't find the link easily to "About this Quick Reference".

**Comment by shawna-slh** [comment]
The presentation in 4 blocks also does not work well for me. (Sharron: "I find the boxes of text hard to look at when displayed and the layout a bit confusing") Also, I think: 1. Some of the sections will be longer, 2. Some of the sections are more important and should have higher visibility. I think Sharing and Contribute should have lower visibility or emphasis. [medium]

**Comment by shawna-slh** [comment]
Thinking about the users, and that we want this to be a primary entry point for WCAG:
- Some people won't know what "Techniques" or "Understanding" is. Many people will.
  Implication: Provide some background information and links to more – probably open by default the first time users come to this page. Make it easily hide-able, and probably not open by default on subsequent visits.
- The functionality will be new to everyone – even people familiar with the old quickref and with WCAG.

[… sorry I didn't get to complete my thoughts. But wanted to put what I have here now …]

@@ need to have some of the info from the previous intro
@@ separate issue: make link to old one still available?
